### PR TITLE
ADDITION: grocy

### PR DESCRIPTION
**Pull request by bcurran3**
**What this PR does / why we need it**:

Adds grocy, the self-hosted groceries and household management solution.

**Which issue (if any) this PR fixes**:

Fixes # n/a

**Any other useful info**:

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -728,6 +731,15 @@ firefly_available_externally: "false"
 firefly_data_directory: "{{ docker_home }}/firefly"
 firefly_port: "8066"
 
+###
+### grocy
+###
+grocy_available_externally: "false"
+grocy_data_directory: "{{ docker_home }}/grocy"
+grocy_user_id: "1000"
```
Comment (davestephens) [inline group_vars/all.yml:739]: Shouldn't this be `0`?

File: group_vars/all.yml [hunk 2]
```diff
@@ -728,6 +731,15 @@ firefly_available_externally: "false"
...
+grocy_group_id: "1000"
```
Comment (davestephens) [inline group_vars/all.yml:740]: And this?

File: tasks/grocy.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    name: grocy
+    image: linuxserver/grocy:latest
+    pull: true
+    volumes:
+      - "{{ grocy_data_directory }}/config:/config:rw"
+    ports:
+      - "{{ grocy_port_http }}:80"
+    env:
+      TZ: "{{ ansible_nas_timezone }}"
+      PUID: "grocy_user_id"
```
Comment (davestephens) [inline tasks/grocy.yml:21]: Looks like some missing `{{ }}` here

File: tasks/grocy.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      PGID: "grocy_group_id"
```
Comment (davestephens) [inline tasks/grocy.yml:22]: And here

File: group_vars/all.yml (hunk 1 above)
Comment (bcurran3) [reply group_vars/all.yml:739]: From what I remember this is one of those containers that you can't run as root due to the web server.

File: group_vars/all.yml (hunk 2 above)
Comment (bcurran3) [reply group_vars/all.yml:740]: From what I remember this is one of those containers that you can't run as root due to the web server.

File: tasks/grocy.yml (hunk 3 above)
Comment (bcurran3) [reply tasks/grocy.yml:21]: Good catch - thanks. Will fix NOW!

File: tasks/grocy.yml (hunk 4 above)
Comment (bcurran3) [reply tasks/grocy.yml:22]: Good catch - thanks. Will fix NOW!

**Comment by bcurran3** [comment]
I think GTG now.

**Comment by bcurran3** [comment]
> Funnily enough I had Grocy open in a tab on my mobile - was nice to see this come in!
> 
> Needs a few tweaks then we should be good to go 👍

Funny. Great minds think alike? Or scary? LOL!

**Comment by davestephens** [comment]
If this is flipped to an Ansible role and tidied up I can review and merge.

**Comment by bcurran3** [comment]
Thanks for the response to my statement saying it was ready to go 834 days ago. 
I'll close now and probably re-submit in < 834 days from now.

**Pull request closed without merging**